toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -179,6 +179,22 @@ SystemConfig:
       Permissions: "664"
 ```
 
+## KernelCommandLine type
+
+Options for configuring the kernel.
+
+### ExtraCommandLine
+
+Additional Linux kernel command line options to add during partition customization.
+
+This value may only be used when when the [Disks](#disks-disk) and
```

Comment:
Would the same limitation apply to EMU? We thought we can leverage the same functionality from EMU for Trident.

Reply:
Yes, this field won't be usable with the OS-modifier. But `CommandLineAdd` below should be usable.